.docker/testing/Dockerfile

```diff
@@ -2,7 +2,7 @@ ARG qfield_test_docker_version
 FROM opengisch/qfield-test-docker:${qfield_test_docker_version}
 MAINTAINER Matthias Kuhn <matthias@opengis.ch>
 
-RUN dnf install -y xorg-x11-server-Xvfb
```

Comment:
Can we put that to osgeo4a?

Reply:
both actually (will shorten build time)

Reply:
It's actually not needed. I was trying to see if that would bring the QML charts into the APK, it did not so I'm planning to revert.

Reply:
ah yeah, this one is pure linux tests, nothing related to apk